examples/jaeger-hotrod/pipelines.yaml

```diff
@@ -16,8 +16,8 @@ raw-pipeline:
     - otel_traces:
   sink:
     - opensearch:
-        hosts: [ "https://node-0.example.com:9200" ]
-        cert: "/usr/share/opensearch-test/root-ca.pem"
+        hosts: [ "https://opensearch:9200" ]
+        insecure: "true"
```

Comment:
As a community, we want to encourage secure best-practices. Thus, we intentionally use TLS hostname validation in this demo. We should continue to use it here.

For this PR, you could either:
1. Go back to using `node-0.example.com` for now.
2. Update the PEM file to include `opensearch`

Reply:
> Go back to using `node-0.example.com` for now.

- I would say I would definitely go with this option, and I have updated the commit and PR descriptions. In most issues, most people have never manually parsed the contents of the TLS certificate. This is exactly what led to a bunch of related issues. I prefer to get `root-ca.pem` directly from the OpenSearch Docker Container.
- I also updated the screenshots in the PR description for the combination of OpenSearch 2.18.0, OpenSearch DashBoards 2.18.0, `otel/opentelemetry-collector-contrib:0.113.0`, and `jaegertracing/example-hotrod:1.63.0`.